releases/v21.2.5.md

```diff
@@ -0,0 +1,150 @@
+- New role `VIEWACTIVITYREDACTED` that works similar as `VIEWACTIVITY` but restricts the usage of the [statements diagnostics bundle](ui-statements-page.html#diagnostics). It is possible for a user to have both roles (`VIEWACTIVITY` and `VIEWACTIVITYREDACTED`), but the role `VIEWACTIVITYREDACTED` takes precedent on restrictions. [#74862][#74862]
+- New role `NOSQLLOGIN` (and its inverse `SQLLOGIN`), which restricts [SQL CLI login](cockroach-sql.html#prerequisites) ability for a user while retaining their ability to log in to the DB Console (as opposed to `NOLOGIN` which restricts both SQL and DB Console). Without any role options all login behavior remains permitted as it does today. OIDC logins to the DB Console continue to be permitted with `NOSQLLOGIN` set. [#75185][#75185]
+- Statements are no longer formatted prior to being sent to the UI, but the new builtin function remains. [#75463][#75463]
+- Added support for SQL arrays containing JSON for in-memory processing. This does not add support for storing SQL arrays of JSON in tables. [#75529][#75529]
+- The default SQL stats flush interval is now 10 minutes. [#75524][#75524]
+
+### Operational changes
+
+- The meaning of `sql.distsql.max_running_flows` cluster setting has been extended so that when the value is negative, it would be multiplied by the number of CPUs on the node to get the maximum number of concurrent remote flows on the node. The default value is -128, meaning that on a 4 CPU machine we will have up to 512 concurrent remote DistSQL flows, but on a 8 CPU machine up to 1024. The previous default was 500. [#75509][#75509]
+
+### Command-line changes
+
+- Not finding the right certs in the certs dir or not specifying a certs dir or certificate path now falls back on checking server CA using Go's TLS code to find the certificates in the OS trust store. If no matching certificate is found, an x509 error will occur announcing that the certificate is signed by an unknown authority. [#74544][#74544]
+- Fixed the CLI help text for `ALTER DATABASE` to show correct options for `ADD REGION` and `DROP REGION`, and include some missing options such as `CONFIGURE ZONE`. [#75067][#75067]
+
+### API endpoint changes
+
+- Created a new endpoint `/sqlroles` that returns a list of the SQL roles for the logged in SQL user. [#75164][#75164]
+
+### DB Console changes
+
+- The **clear SQL stats** links on the statement and transaction pages were relabeled **reset SQL stats**, for consistency with the language in the SQL shell. [#74417][#74417]
+- The terminate session and terminate query buttons are available to be enabled in the cluster UI. [#74530][#74530]
+- Replaced raw statement with statement ID in the URL. [#75463][#75463]
+- Removed `$ internal` as one of the apps option under the Statements and Transactions page filters. [#75485][#75485]
+- Changed the order of tabs under SQL Activity page to be Statements, Transactions, and Sessions. [#75503][#75503]
```

Comment:
There isn't because it was just added to group together Statements, Transactions, and Sessions, and those existing pages were updated to say that you need to click SQL Activity first. I don't think we need a separate SQL Activity doc.

Reply:
Changed my mind. Added section to `ui-overview.html`.